### ISSUE TITLE
The page doesn't alterate when I change the informations of the tasks

### ISSUE THREAD

**Issue by JoaoPaulo-LD** [JoaoPaulo-LD/taskfocus-todo#1]
### Big issue
When I change informations in the PUT method on the "AlterateTask" page, this isn't reflected by the page where the task is located. The changes only appear if I exit the page and entry again. I think that is a issue in the useEffect to load the tasks. I'm studying about useCallback and I hope that this resolve my problems.

**Comment by JoaoPaulo-LD** [comment]
`useEffect(() => {
        loadTasks()
    }, [tasks])`
When I put the state to "reload" the screen without leaving the page the application goes into a loop...

**Comment by joaorodrs** [comment]
I solved the problem by adding "useIsFocused" from "react-navigation / native". This hook has a Boolean value and if the screen where it is inserted is showing it will have the true value.

`import { useIsFocused } from 'react-navigation/native'`

`const isFocused = useIsFocused()`

Therefore, I defined that every time the user enters the screen, the "useEffect" is called again. For that I put a variable with "useIsFocused" in the dependency array of "useEffect", when the user enters the screen the isFocused will be set to true and the useEffect will know that this constant changes.
```
useEffect(() => {
    // load the tasks from api when user open this screen
}, [ isFocused ])
```
When the user enters the screen, the task will load and will show updated in the screen.
